### DISCUSSION TITLE
Hyperion - HyperHDR change was successful.

### DISCUSSION THREAD

**Discussion by bong8839** [awawa-dev/HyperHDR#207] [Q&A]
Hi. Thanks to your guidance, first of all, system changes have been completed. thank you so much.
There was a part missing from the picture last time, so I will correct it and re-upload it so that beginners like me can easily recognize it.

I uploaded a photo.

![하이퍼 HDR 그림](https://user-images.githubusercontent.com/97230290/152601798-617acb10-fe16-4cd3-8230-7c3f21a9feea.png)

I'm running some tests right now, and I have a question.
1. If you select Auto Signal Detection in the video capture menu, the LED may turn off due to no signal on the TV background screen. I don't know if it's been the wallpaper for a long time, but is it normal to play a video and then turn off after a few seconds?

2. I checked in the video device information, my capture card is 1280x720, yuv 10 frames and mjpeg 60 frames are available. But in the settings, it is possible to set up to yuv 60 frames, wouldn't it be meaningless? Is it correct to just set mjpeg 60 frames?

I can still feel the very slight delay, is there any recommended resolution or frame rate?

3. If you look at the log, frame information is displayed as shown below. Is it okay?
<2022-02-04T20:01:24.873Z [V4L2:AUTO] Video FPS: 60.03, av. delay: 6ms, good: 3602, bad: 0 (60.00,15)>

**Comment by awawa-dev** [comment]
> Hi. Thanks to your guidance, first of all, system changes have been completed. thank you so much. There was a part missing from the picture last time, so I will correct it and re-upload it so that beginners like me can easily recognize it.
> 
> I'm running some tests right now, and I have a question.
> 
>     1. If you select Auto Signal Detection in the video capture menu, the LED may turn off due to no signal on the TV background screen. I don't know if it's been the wallpaper for a long time, but is it normal to play a video and then turn off after a few seconds?

Not sure if it is automatic signal detection..what is in the logs when that happens and did you calibrated the auto-detection using grabber no signal board (for example after disconnected HDMi cable from the grabber)? Was the calibration succesful?

>     2. I checked in the video device information, my capture card is 1280x720, yuv 10 frames and mjpeg 60 frames are available. But in the settings, it is possible to set up to yuv 60 frames, wouldn't it be meaningless? Is it correct to just set mjpeg 60 frames?
>

It's explained in v17 blog article, last entry (also read about the auto-calibration):
https://www.hyperhdr.eu/2021/08/hyperhdr-v17beta-preview.html

Combos allows you to select "wrong" combination. They were left for backward compatibility. That's why you should use the new form to select resolution/color format/refresh rate. The grabber won't you allow to activate unsupported combination anyway. I guess you ended up with 60FPS MJPEG (there is a message about selected mode in the logs during grabber initialization or in the overview tab). 

> 
> I can still feel the very slight delay, is there any recommended resolution or frame rate?
> 
>     3. If you look at the log, frame information is displayed as shown below. Is it okay?
>        <2022-02-04T20:01:24.873Z [V4L2:AUTO] Video FPS: 60.03, av. delay: 6ms, good: 3602, bad: 0 (60.00,15)>

Yes, pro-tips rules: https://github.com/awawa-dev/HyperHDR/wiki/Quick-start
Best to select resolution around 640x480/60Hz or similar depending on your grabber. Enabling quarter of frame mode gives additional boost.

**Reply by bong8839** [reply]
Hi. Operational tests have been completed.

A problem has been identified.

1. If Quarter of frame mode is not used, the LED does not turn off with a weak white light even when the TV is turned off. When in use, it turns off after no signal detection time.

2. In the normal black level or where no color is detected, the white LED is continuously lit. Is this an editable option?

3. If you set HDR to SDR tone mapping, check the color carefully when playing HDR video or general video. The problem is that when you use this setting, the colors cannot be detected properly when viewing photos in the gallery. If set to Disabled, color detection in HDR videos is not properly performed, and colors are detected well in gallery photos. In the past, when using Hyperion, there was no separate HDR tone mapping function, but color detection was good for HDR videos, general videos, and gallery photos.

**Reply by awawa-dev** [reply]
There is 'minimal backlit' option in the processing tab. It causes LED to lit even if the input is black. Maybe it's enabled? USB grabber almost never provide real black color that would case the LED to turn off: it's very dark grey instead, it causes the LED strip to lit even if you see the black on the TV screen. There is no auto-detection for HDR & SDR: but usually it can be force in the system or the video player to use HDR output only.: then HyperHDR option 'HDR tone mapping' can be always enabled.

**Reply by awawa-dev** [reply]
If you enable or disable 'Quarter of frame mode' then you MUST calibrate auto-detection again. Otherwise it won't work properly.

**Reply by bong8839** [reply]
Well.  okay.  Let's test it again today.  Do you know of a case where a certain grabber sends a no-signal image when the signal detection option is selected and the TV is turned off?  This seems to happen when the light turns on for a while when the TV is turned off.

**Reply by awawa-dev** [reply]
>  Do you know of a case where a certain grabber sends a no-signal image when the signal detection option is selected and the TV is turned off?

I don't think I understand it correctly. The grabber's no-signal board is not linked to a TV that is on or off, but to an grabber's HDMI source if available or not . Again, in your HyperHDR's log you can find the reason why LEDs were turned on or off.

**Reply by bong8839** [reply]
Since I use a translator to communicate, I can't seem to explain it properly.  sorry.  I'll try the test again later and post a video or something like this and talk about it, I think it'll be easy to understand.

**Reply by awawa-dev** [reply]
OK, English is not my primary either. Verify the HyperHDR log first as no one else will do it for you . And grabber's no-signal board has nothing to do with whether the TV is turned on or off.
